crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/util/SortUtilsTest.java

```diff
@@ -16,4 +16,11 @@ public void testSortUtilConstructorIsPrivate() throws NoSuchMethodException {
         Constructor<SortUtils> constructor = SortUtils.class.getDeclaredConstructor();
         assertTrue(Modifier.isPrivate(constructor.getModifiers()));
     }
+
+    @Test
+    public void testSort_whenNullIsPassed_NullIsReturned() {
+        List<String> list = null;
+        SortUtils.sortListWithComparing(list, null, null, null);
+        assertNull(list);
```

Comment:
Can we make sure we're using AssertJ fluent asserts here e.g. assertThat().isNull(). The majority of our other test classes do this and we should be aiming to maintain consistency.